test/invocation/block_argument_kind.stmt

```diff
@@ -25,46 +25,18 @@ function((parameter) { body; });
 function((parameter) {
   body;
 });
->>> Empty block-bodied function expression with parameters.
+>>> Empty block-bodied function expression with parameters is not a block argument.
 function((parameter, anotherParameter) {});
 <<<
-function((
-  parameter,
-  anotherParameter,
-) {});
->>> Expression-bodied function expression with parameters.
+function(
+  (parameter, anotherParameter) {},
+);
+>>> Expression-bodied function expression with parameters is not a block argument.
 function((parameter, anotherParameter) => body);
 <<<
-function((
-  parameter,
-  anotherParameter,
-) => body);
->>> Block-bodied function expression with block comment in the parameters.
-function((/* very long block comment */) {});
-<<<
-function((
-  /* very long block comment */
-) {});
->>> Block-bodied function expression with line comment in the parameters.
-function((// comment
-) {});
-<<<
-function((
-  // comment
-) {});
->>> Expression-bodied function expression with block comment in the parameters.
-function((/* very long block comment */) => body);
-<<<
-function((
-  /* very long block comment */
-) => body);
->>> Expression-bodied function expression with line comment in the parameters.
-function((// comment
-) => body);
-<<<
-function((
-  // comment
-) => body);
+function(
+  (parameter, anotherParameter) => body,
```

Comment:
Can you add a testing example (if it's not already here, I didn't check too hard!) with a parameter list that's > 40 characters? Would we expect it to block split then?

Reply:
Ah, testing that's a good idea, thanks!

It doesn't block split then either. You get:

```dart
function(
  (
    parameter,
    anotherParameter,
    thirdParameter,
  ) => body,
);
```

I think I prefer that over block splitting it (and in practice, it's rare for a lambda to have so many parameters that they split).